<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QToolButton" name="forwardButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>210</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Downloads/SPARKI Icons/Alarm-Arrow-Up-icon.png</normaloff>../../../../Downloads/SPARKI Icons/Alarm-Arrow-Up-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>100</number>
    </property>
   </widget>
   <widget class="QToolButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>310</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Downloads/SPARKI Icons/Alarm-Arrow-Down-icon.png</normaloff>../../../../Downloads/SPARKI Icons/Alarm-Arrow-Down-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="rightButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>310</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Downloads/SPARKI Icons/Alarm-Arrow-Right-icon.png</normaloff>../../../../Downloads/SPARKI Icons/Alarm-Arrow-Right-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="leftButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>310</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Downloads/SPARKI Icons/Alarm-Arrow-Left-icon.png</normaloff>../../../../Downloads/SPARKI Icons/Alarm-Arrow-Left-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>NULL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed (50-250)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Command Sent</string>
    </property>
   </widget>
   <widget class="QPushButton" name="transmitButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Transmit</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>100</number>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>677</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuS_P_A_R_K_I">
    <property name="title">
     <string>S.P.A.R.K.I.</string>
    </property>
   </widget>
   <addaction name="menuS_P_A_R_K_I"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
